src/vulkan/command.cc

```diff
@@ -108,7 +108,7 @@ Result CommandBuffer::SubmitAndReset(uint32_t timeout_ms) {
     return Result("Vulkan::Calling vkQueueSubmit Fail");
 
   VkResult r = vkWaitForFences(device_, 1, &fence_, VK_TRUE,
-                               timeout_ms * 1000 * 1000 /* nanosecond */);
+                               timeout_ms * 1000ULL * 1000ULL /* nanosecond */);
```

Comment:
It may be fine but explicit type conversion for `timeout_ms` would be nice.
Could you please add `static_cast<uint64_t>` for `timeout_ms`?

i.e.,

```
static_cast<uint64_t>(timeout_ms) * 1000ULL * 1000ULL /* nanosecond */);
```